en/config/actuators.md

```diff
@@ -33,50 +33,87 @@ This particular frame also includes an image showing the motor positions, which
+It is extremely important that we decouple the concept of Control Surface, Servo and Motor Tilt directions in Control Allocation from the actual Servo output / connections in the physical vehicle.
+
+That means that the Control Allocation Control Surface & Servo settings are physical vehicle independent. Therefore when we think about the Surface Movements, we imagine the ideal vehicle as shown below that moves actuators in the given direction!
 :::
 
+![Control Surface Deflections](../../assets/config/actuators/plane_control_surface_convention.png)
+
+Control surfaces use the following deflection direction convention:
+
+#### Horizontal Control Surface (e.g. Aileron)
+> Upwards movement equals Positive deflection
+
+'Pitch UP' control input should command positive deflection to both ailerons
+
+* Example : To pitch in positive direction (pitching skywards), 'Both Ailerons (Right & Left)' need a positive deflection.
+
+'Roll RIGHT' control input should command positive deflection to right aileron and negative deflection to left aileron
+
+* Example : To roll in positive direction (banking to the right), the 'Right Aileron' needs a positive deflection.
+
+#### Vertical Control Surface (e.g. Rudder)
+> Rightwards movement equals Positive Deflection
+
+'Yaw RIGHT' control input should command positive deflection.
+
+* Example : To yaw in positive direction (clockwise rotation), conventional rudder airplane needs a positive deflection.
+
+#### Mixed Control Surface (e.g. V-Tail)
+> Upwards movement equals Positive deflection
+
+'Pitch UP' control input should command positive deflection to both V-tails
+
+* Example : To pitch in positive direction (pitching upwards),  'Both V-tail surfaces' need a positive deflection.
+
+> Rightwards movement equals Positive Deflection
+'Yaw RIGHT' control input should command positive deflection to both V-tails.
+
+* Example : Having a V or A shaped tail in the back, positive deflection in Rightwards direction will Yaw the vehicle to the right.
+
+
+### Motor Tilt Servos (Conceptual)
+![Tilt Servo Geometry Setup Example](../../assets/config/actuators/tilt_servo_geometry_config.png)
+
+> Example Tilt Servo Geometry setup for a Tiltrotor VTOL
+
+Similar to the Control Surfaces, Servos can also rotate the motors themselves! This is often the case in a VTOL like [Tiltrotors](/en/frames_vtol/tiltrotor.md). 
 
-### Motor Tilt Servos
+![Tilt Axis](../../assets/config/actuators/tilt_axis.png)
 
-Tilt servos are configured as follows:
-- The reference direction is upwards (negative Z direction).
-- Tilt direction: **Towards Front** means the servo tilts towards positive X direction, whereas **Towards Right** means towards positive Y direction.
-- Minimum and maximum tilt angles: specify the physical limits in degrees of the tilt at minimum control and maximum respectively.
-  An angle of 0° means to point upwards, then increases towards the tilt direction.
```

Comment:
Why did you remove this?

Reply:
Either way I don't understand the way this works/coordinate system. 

My understanding of tiltrotors comes from the eflight convergence. This has a rotor that points up to hover and forward to fly forward. My understanding is that this would be set to "Towards Front". It points either forward or straight up but it isn't clear to me what the min and max tilt angle will be - I know one will be 0, but not which position, and also whether the other value will be 90 or -90.
Also, what does it mean if the motor goes to the right? Does that mean you can't use it as a pusher?

Reply:
There's a lot of degrees of freedom here, hence it's complicated. You can think of the tilt rotating within a vertical plane, with the 'yaw' of the plane given by the tilt direction (front or right in that case).
Then within the plan you have a starting angle and an end angle. You can think of a clock: 12=upwards=0 deg=always aligned with Z axis, then the angle increases clockwise (if you were to look at the plane from the right side).
Not sure if that helps or makes it even more confusing.

Reply:
For the eflight I can kind of see this. The min angle is 0 when in hover, and the max angle is 90 and the tilt is forward. From the right I can see the motor moving from its spindle being pointed up to being forwards, a move from 12 oclock (0) to 3oclock (90). 

But if the motor was moving right then I would see the props coming from invisible to visible and facing me. I guess the move is still 0 to 90. But what use is it? When facing me it is pulling the VTOL "plane" sideways?

IF I had an eflight that used a prop like a pusher below the vehicle and then moved forward, the angle would change from -180 to 90 anticlockwise (from right). 

Nup. I guess I don't get it :-(. I do know this does not pass the "idiot gets it test"